vendor/github.com/knative/serving/pkg/reconciler/v1alpha1/clusteringress/config/testdata/config-istio.yaml

```diff
@@ -0,0 +1 @@
+../../../../../../config/config-istio.yaml
```

Comment:
Not sure what this is!

Reply:
This is pulled from dep and it is specified in https://github.com/knative/serving/blob/master/pkg/reconciler/v1alpha1/clusteringress/config/testdata/config-istio.yaml

Where they are using the relative path to pull in the yaml file.